impl/src/main/java/org/jboss/cdi/tck/tests/se/context/activation/interceptor/MethodInterceptorContextActivator.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2016, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.cdi.tck.tests.se.context.activation.interceptor;
+
+import javax.enterprise.context.control.ActivateRequestContext;
+import javax.inject.Inject;
+
+@ActivateRequestContext
```

Comment:
If you want a method interceptor, shouldn't this annotation be only on the given method?